apps/backend/api/src/app.module.ts

```diff
@@ -19,10 +26,13 @@ import { AppService } from './app.service';
         password: configService.get('DATABASE_PASSWORD'),
         database: configService.get('DATABASE_NAME'),
         synchronize: /true/.test(configService.get('DATABASE_SYNCHRONIZE')),
-        entities: [],
+        entities: entities,
```

Comment:
Quand le nom de la prop est identique au nom de la variable qui 
```suggestion
        entities,
```